src/API/Google/Settings.php

```diff
@@ -42,6 +42,26 @@ public function __construct( ContainerInterface $container ) {
 		$this->container = $container;
 	}
 
+	/**
+	 * Return a set of formatted settings which can be used in tracking.
+	 *
+	 * @since x.x.x
+	 *
+	 * @return array
+	 */
+	public function get_tracked_settings() {
```

Comment:
`get_tracking_settings` implies it would be settings for the tracking system, which is not the case.

Changing the tense of the verb to "tracked" implies that it is "for tracking", but happy to change it to `get_settings_for_tracking` if it's more explicit.